client/client.go

```diff
@@ -61,6 +61,7 @@ type RequestOptions struct {
 
 type RequestResponse struct {
 	StatusCode int
+	Header     http.Header
```

Comment:
Sorry to nitpick, but I think I'd like this called `Headers` plural to match the field name in `RequestOptions` (I've always found it weird that the `net/http` field name is singular anyway).